core/model/src/main/java/org/eclipse/rdf4j/model/util/RDFContainers.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+package org.eclipse.rdf4j.model.util;
+
+import org.eclipse.rdf4j.model.*;
```

Comment:
Our formatting guidelines don't allow wildcard imports. Have a look at https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#code-formatting for some tips on how set up your environment and/or run a few quick maven commands to fix your formatting.

Reply:
We will take care about this in our further commits/PRs.

Reply:
You'll also need to run `mvn impsort:sort`.

Reply:
`mvn impsort:sort` has fixed the formatting successfully.